santa/santa.go

```diff
@@ -16,22 +16,22 @@ type Config struct {
 }
 
 // Rule is a Santa rule.
-// Full documentation: https://github.com/google/santa/blob/01df4623c7c534568ca3d310129455ff71cc3eef/Docs/details/rules.md
+// Full documentation: https://github.com/google/santa/blob/main/docs/concepts/rules.md
 type Rule struct {
 	RuleType      RuleType `json:"rule_type" toml:"rule_type"`
 	Policy        Policy   `json:"policy" toml:"policy"`
-	SHA256        string   `json:"sha256" toml:"sha256"`
+	Identifier    string   `json:"identifier" toml:"identifier"`
```

Comment:
Have you tested this with the latest Santa client? Last time I did the Santa client couldn't handle extra rules entries. 

For example, if we send back a BINARY rule with a SHA256 entry and an empty Identifier (and vice versa), it will crash when trying to process the rules.

Example (bad):
```
{ "rules": [ { "rule_type": "BINARY", "policy": "BLOCKLIST", "sha256": "2dc104631939b4bdf5d6bccab76e166e37fe5e1605340cf68dab919df58b8eda", "identifier": "", "custom_msg": "blocklist firefox" }, { "rule_type": "TEAMID", "policy": "ALLOWLIST", "sha256": "", "identifier": "EQHXZ8M8AV", "custom_msg": "allow google team id" } ] }
```

Santa client has this error:
```
Received 2 rules Failed to add rule(s) to database: Rule array contained invalid entry: SNTRule: Identifier: , State: 2, Type: 1, Timestamp: 0 Failure reason: SNTRule: Identifier: , State: 2, Type: 1, Timestamp: 0
```